pkg/controller/perconaxtradbcluster/perconaxtradbcluster_controller.go

```diff
@@ -264,6 +286,26 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 		if err != nil && !errors.IsAlreadyExists(err) {
 			return fmt.Errorf("create PXC Service: %v", err)
 		}
+
+		// PodDistributedBudget object for ProxySQL
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: proxySet.Name, Namespace: proxySet.Namespace}, proxySet)
+		if err != nil {
+			if !errors.IsNotFound(err) {
+				return fmt.Errorf("Get stateful set %s: %v", proxySet.Name, err)
```

Comment:
Error strings should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
and I'm not sure that `proxySet.Name` is safe to use here. You sure in can't a nil after Get returns with error? Just put "get proxy stateful set: %v"
\* _the same above_